aws-csharp/Aws.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pulumi" Version="1.5.0-*" />
+    <PackageReference Include="Pulumi.Aws" Version="1.8.0-*" />
```

Comment:
Is there a way to make this float more permissively? Or do we need to come back and bump this for every new minor/revision release we make?

Reply:
We can make it `1-*` but this currently doesn't work because of a bit of a mess in our current packages. E.g. [Pulumi.Aws](https://www.nuget.org/packages/Pulumi.Aws/1.8.0-preview-alpha.1572536463) depends on `Pulumi (>= 1.5.0-preview-alpha-nugetTest.1572398968) ` which is not listed but still is the highest version semver-wise. This breaks `1-*` resolve. I brought this up in slack.

Reply:
Added